<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>361</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>60</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Permutation Symmetry</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Max Order</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Data File Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="button1">
     <property name="geometry">
      <rect>
       <x>219</x>
       <y>240</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate Basis</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>361</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Num Atoms</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Num Configurations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Polynomial Order</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Symmetry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Num Coefficients</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>190</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>731</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Weights</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>561</width>
       <height>88</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>We can apply weight in the fitting, and the weight of each point is given by wt = E0/(E0+dE),
where dE is the energy of that point relative to the minimum. E0 is a user-specified parameter
(in unit Hartree).</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>If you would like to apply the weight, please input the E0 parameter. Otherwise please
enter &quot;n&quot;.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>40</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button4">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>70</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>731</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Parameters</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>561</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Please specify the a0 parameter (in unit Bohr) used in Morse variable yij = exp(-rij/a0). 
The recommended range is 2.0-3.0 Bohr.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button5">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>50</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>600</y>
      <width>491</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Fitting</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>160</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Overall RMSE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Weighted RMSE</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button6">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Compute</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button7">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>275</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>600</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Reset</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Resets directory. Does not save.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="button8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>731</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Read</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>711</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Please make sure Perl, Python, one Fortran compiler (e.g., ifort, GFortran), one C++ compiler, and one linear algebra 
ibrary (e.g. MKL, AOCL) are installed and loaded, and then modify the Makefiles accordingly. Refer to ReadMe for
software requirements.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
